### PR TITLE
Add Tests Against Node 8

### PR DESCRIPTION
**Pull request by mhoeger**
**NOTE: do not merge until conditional logic on nightly builds added!**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -62,3 +62,5 @@ for:
   test_script:
   - ps: >
       .\run-tests.ps1
+      
+      .\run-tests.ps1 -NodeVersion "8.11.1"
```
Comment (fabiocav) [inline appveyor.yml]: This is triggering a full test run for all test projects and categories, which is unnecessary and has significant impact on build times. Can we scope this down to re-run only the relevant tests?

Comment (mhoeger) [reply appveyor.yml]: Yes! I was also thinking of only running on nightly builds? But let me know what you think

Comment (mhoeger) [reply appveyor.yml]: Ideally the bulk of the node testing will be here: https://github.com/Azure/azure-functions-nodejs-worker/pull/100

That's why I was thinking adding nightly testing might be good? Just as an extra layer of assurance that doesn't need to block or slow down normal development

**Comment by mathewc** [comment]
What is the status of this PR?

**Pull request closed without merging**